pkg/doc/verifiable/credential_jwt.go

```diff
@@ -17,7 +18,7 @@ import (
 type JWTCredClaims struct {
 	*jwt.Claims
 
-	Credential *rawCredential `json:"vc,omitempty"`
+	VC map[string]interface{} `json:"vc,omitempty"`
```

Comment:
Is this field intended to be used outside of the package?

Reply:
We cannot make it unexported as **json** won't take it into account on serialization & deserialization.

Reply:
Is the struct used outside of the package?

Reply:
Yes, it could.